lib/managedservicesplatform/runtime/contract/postgresql.go

```diff
@@ -99,5 +116,63 @@ func parseCustomDSNTemplateConnConfig(customDSNTemplate, database string) (*pgxp
 	if err != nil {
 		return nil, errors.Wrap(err, "rendered PGDSN is invalid")
 	}
+
+	if logQueries {
+		logger := log.Scoped(fmt.Sprintf("pgx.devtracer.%s", database))
+		logger.Warn("enabling query logging")
```

Comment:
```suggestion
		logger.Info("enabling query logging")
```

Given this always local dev, I feel like Info is more appropriate.